blocks/feed/feed.js

```diff
@@ -141,6 +143,14 @@ export default async function decorate(block) {
   block.innerHTML = '';
   const blockContents = resultParsers[blockType](results, blockCfg);
   const builtBlock = buildBlock(blockType, blockContents);
+  if (feedTags && blockContents.length) {
+    const imageThumbnail = blockContents[0][0];
+    if (imageThumbnail) {
+      setOgImages(imageThumbnail);
+      setMetaTag('meta', 'property', 'og:url', `${window.location.href}`);
+      setMetaTag('link', 'rel', 'canonical', `${window.location.href}`);
```

Comment:
Why would these two meta tags not be set if `imageThumbnail` is missing?

Reply:
Good catch!